packages/client-core/src/admin/services/AvatarService.ts

```diff
@@ -73,9 +74,24 @@ export const AvatarService = {
       dispatch(AvatarAction.avatarsFetched(avatars))
     }
   },
-  createAdminAvatar: async (data: any) => {
+  createAdminAvatar: async (blob: Blob, thumbnail: Blob, data: any) => {
     const dispatch = useDispatch()
     try {
+      if (blob) {
+        const uploadArguments: AdminAssetUploadType = {
+          type: 'admin-file-upload',
+          files: [blob, thumbnail],
+          args: [
+            {
+              key: `avatars/public/${thumbnail['name']}`,
```

Comment:
to make it more uniwue you can append unicode timestamp with file name as well

Reply:
I made changes from backend, That is where i modify key property